Resources/ServerInfo/Guidebook/Engineering/TEG.xml

```diff
@@ -21,19 +21,153 @@
+  To increase the output's airflow, pipe as many air scrubbers as the room can hold to the output. You can't stack scrubbers, so be efficient!
+
+  Optionally, you can use a network configurator to link all the air scrubbers to the air alarm present on some burn chambers. If you want to extract tritium, do this and set the scrubbers to filter for tritium gas.
+  If not, it is still recommended to do this step, and set the gas scrubbers to scrub everything but Plasma and O2, and also set the scrubbers to siphon air instead. This ensures as much hot, non-burnable air is pulled in as possible.
+
+  # The Cold Loop
 
-  As I'm sure a wise person once said: the best way to make something hot is to light it on fire. Well, depending on context that may not be very wise, but luckily your engineering department has just what's needed to do it wisely after all.
+  As with the Hot Loop, the Cold Loop must also be setup in order to operate the TEG. However, the Cold Loop is usually a lot more low-tech than the Hot Loop; in reality, the Cold Loop only has to be "relatively" cooler -- hey, room temperature is technically cooler than the surface of the sun, right?
 
-  TODO: somebody fill this out once we settle on a general mapped layout for burn chambers.
+  There are 3 main methods you will see used for the Cold Loop: The Water Cooler (see: Liltenhead's video on the TEG), the Coolant Array and the Freezer Loop.
+
+  ## The Water Cooler
+
+  An equally naive method as the Pipe Burn, this simply involves taking some useless gas (in this case, Water Vapour) and flowing it through the TEG and into space. It's dirt-cheap and simple, at the cost of efficiency and permanant loss of gas.
+
+  Setting this up is so simple, even Hamlet could manage it. Just take an output of a gas (here, Water Vapour), send it through the Cold side of the TEG, and then vent it into space.
+
+  <Box>
+    <GuideEntityEmbed Entity="GasPressurePump" Rotation="90" Caption="" Margin="-1"/>
+    <GuideEntityEmbed Entity="GasPipeStraight" Rotation="90" Caption="" Margin="-1"/>
+    <GuideEntityEmbed Entity="TegCirculator" Rotation="90" Caption="" Margin="-1"/>
+    <GuideEntityEmbed Entity="GasPipeStraight" Rotation="90" Caption="" Margin="-1"/>
+    <GuideEntityEmbed Entity="WallReinforced" Rotation="0" Caption="" Margin="0"/>
+    <GuideEntityEmbed Entity="GasPassiveVent" Rotation="270" Caption="" Margin="-1"/>
+  </Box>
+
+  TODO: Remove this section when gas miners are removed in a future atmos refactor.
 
   ## Coolant Array
 
-  A whole bunch of heat-exchanger piping in space. There's not much to say here: gas goes through and cools down.
+  This is the default method for the Cold Loop you will see on a variety of stations. Being of moderate complexity and having no losses of any resource, this [italics]should[/italics] be the main method of cooling down the TEG. However, space heat exchange kinda sucks at the moment, so this is not as relevant as it should be.
```

Comment:
this is because every map doesnt use radiators properly
to use radiators for cooling you need to have them in parallel, with gas pumps after at max pressure and volume pumps before, set to `200 / number of radiators`
with even 6 of them like this it can keep the cold loop at -60 or something permanently

Reply:
this can also be used for the hot loop, yes? (just checking)
also why has no-one fixed the mapped radiators yet?

Reply:
mappers dont have to be atmos pros so the setup always needs reworking for peak performance

it *can* be used for hot loop but theres no need since its only to increase radiation efficiency, hot loop to burn chamber is just heat transfer and has high heat automatically so T^4 kicks in

Reply:
mmm, should've included that as a detail before this got merged
oh well, if its efficient enough it should be fine (and maybe some common sense on the player's side will help too)